### PR TITLE
Add javascript build

### PR DESCRIPTION
**Pull request by tgjonestx**
https://jira.zenoss.com/browse/ZEN-23963

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+
+build: javascript
+	tar cvfz $(ARTIFACT) Products bin
+
+javascript:
+	make -f javascript.mk build
+
+clean:
+	rm -f $(ARTIFACT)
+	-make -f javascript.mk clean
```
Comment (jplouis) [inline makefile]: is that "-" intentional?

Comment (rDr4g0n) [inline makefile]: I'm guessing that if you try to clean and the files to be cleaned are absent it will exit non-zero, so the dash says to ignore that.

**Comment by rDr4g0n** [comment]
I'm good with this if JP is.

File: makefile (hunk 1 above)
Comment (tgjonestx) [inline makefile]: Yes - the dash says - don't exit with an error on this step if it fails.  On Jenkins builds, we always want to execute `make clean build`, though some build slaves may not have any files to clean

**Pull request merged**